core/server/worker/src/main/java/alluxio/worker/block/meta/StorageTierView.java

```diff
@@ -25,29 +23,36 @@
  * This class is a wrapper of {@link StorageTier} to provide more limited access.
  */
 @ThreadSafe
-public final class StorageTierView {
+public class StorageTierView {
 
   /** The {@link StorageTier} this view is derived from. */
-  private final StorageTier mTier;
+  final StorageTier mTier;
   /** A list of {@link StorageDirView} under this StorageTierView. */
-  private final List<StorageDirView> mDirViews = new ArrayList<>();
-  /** The {@link BlockMetadataManagerView} this {@link StorageTierView} is under. */
-  private final BlockMetadataManagerView mManagerView;
+  final List<StorageDirView> mDirViews = new ArrayList<>();
 
   /**
-   * Creates a {@link StorageTierView} using the actual {@link StorageTier} and the above
-   * {@link BlockMetadataManagerView}.
+   * Creates a {@link StorageTierView} using the actual {@link StorageTier}.
    *
    * @param tier which the tierView is constructed from
-   * @param view the {@link BlockMetadataManagerView} this tierView is associated with
    */
-  public StorageTierView(StorageTier tier, BlockMetadataManagerView view) {
+  public StorageTierView(StorageTier tier) {
+    this(tier, false);
+  }
+
+  /**
+   * Creates a {@link StorageTierView} using the actual {@link StorageTier}.
+   *
+   * @param tier which the tierView is constructed from
+   * @param evictable whether this class contains evict-related methods
+   */
+  public StorageTierView(StorageTier tier, boolean evictable) {
```

Comment:
@madanadit Are there better ways than using a boolean?
Here i use `evictable` is because the `StorageDirView` and `StorageDirEvictableView` have different parameters.